translations/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+find_package(Qt5 COMPONENTS Widgets LinguistTools)
+
+set(ghostwriter_TRANSLATIONS
+  ghostwriter_ar.ts
+  ghostwriter_cs.ts
+  ghostwriter_de.ts
+  ghostwriter_en.ts
+  ghostwriter_es.ts
+  ghostwriter_fr.ts
+  ghostwriter_it.ts
+  ghostwriter_ja.ts
+  ghostwriter_nl.ts
+  ghostwriter_pl.ts
+  ghostwriter_pt_BR.ts
+  ghostwriter_ru.ts
+  ghostwriter_zh.ts
+)
+
+qt5_add_translation(QM_FILES
+  ${ghostwriter_TRANSLATIONS}
+)
+add_custom_target(add_translations ALL DEPENDS ${QM_FILES})
+
+qt5_create_translation(QM_FILE_UPDATES
+  ${CMAKE_SOURCE_DIR}/src
+  ${ghostwriter_TRANSLATIONS}
+)
+add_custom_target(update_translations DEPENDS ${QM_FILE_UPDATES})
```

Comment:
The targets here are primarily for dependency management. `add_translations` is marked `ALL` so it will be run every build, since all it does is ensure the compiled translations exist in the build dir. `update_translations` will respond to changes in the source tree when needed. It _should_ run automatically, but can also be triggered manually with a `cmake --build <build_dir> --target update_translations`